tests/parameter_sets/test_example.py

```diff
@@ -0,0 +1,69 @@
+from io import StringIO, BytesIO
+from unittest.mock import patch, Mock
+
+import pytest
+
+from ewatercycle import CFG
+from ewatercycle.parameter_sets import ExampleParameterSet
+
+
+@pytest.fixture
+def setup_config(tmp_path):
+    CFG['parameterset_dir'] = tmp_path
+    yield CFG
+    # Rollback changes made to CFG by tests
+    CFG.reload()
+
+
+class TestGoldenPath:
```

Comment:
What is this?

Reply:
I think I started with the idea of having tests for the happy code path grouped together and other tests for when things go wrong. But seems I did not followed through. Flattened tests again